packages/admin/resources/views/components/stats/card.blade.php

```diff
@@ -138,11 +139,15 @@ class="h-4 w-4"
                     this.chart.data.datasets[0].data = this.values
                     this.chart.update()
                 },
+
+                updateChartColors: function() {
+                    chart = Chart.getChart(this.$refs.canvas);
+                    chart.data.datasets[0].backgroundColor = getComputedStyle($refs.backgroundColorElement).color
+                    chart.data.datasets[0].borderColor = getComputedStyle($refs.borderColorElement).color
+                    chart.update('none');
```

Comment:
I fail to understand why we shouldn't use `this.` here.
Could you clarify, @leandrocfe?

Reply:
> I fail to understand why we shouldn't use `this.` here. Could you clarify, @leandrocfe?

Hi @zepfietje,

The 'this.chart' property can only be accessed during the first initialization. Since the x-data is rendered at each interval, we end up running the 'init' method repeatedly, which resets 'this.chart' to null every time. While you could use 'this.chart' in this scenario, it would involve destroying and recreating the chart at each interval.

Reply:
Thanks for elaborating, @leandrocfe.
I've cleaned up the code via the GitHub UI, so I'm not 100% sure everything still works as expected.

Could you do a final round of testing to see if nothing's broken?
So updating the chart by polling and switching dark theme.